[2025-06-17 15:47:23,672] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.thrid_task 2022-01-17T00:00:00+00:00 [queued]>
[2025-06-17 15:47:23,874] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.thrid_task 2022-01-17T00:00:00+00:00 [queued]>
[2025-06-17 15:47:23,876] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2025-06-17 15:47:23,877] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2025-06-17 15:47:23,884] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2025-06-17 15:47:24,007] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): thrid_task> on 2022-01-17T00:00:00+00:00
[2025-06-17 15:47:24,029] {standard_task_runner.py:52} INFO - Started process 3060 to run task
[2025-06-17 15:47:24,051] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v5', 'thrid_task', '2022-01-17T00:00:00+00:00', '--job-id', '1053', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp_y2xrbxp', '--error-file', '/tmp/tmpweytxz4n']
[2025-06-17 15:47:24,083] {standard_task_runner.py:77} INFO - Job 1053: Subtask thrid_task
[2025-06-17 15:47:24,418] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v5.thrid_task 2022-01-17T00:00:00+00:00 [running]> on host 4f8b76136637
[2025-06-17 15:47:24,647] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=thrid_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-17T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-17T00:00:00+00:00
[2025-06-17 15:47:24,648] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2025-06-17 15:47:24,667] {bash.py:158} INFO - Running command: echo hey, I am task3 and will be running after task1 at the same time as task2!
[2025-06-17 15:47:24,755] {bash.py:169} INFO - Output:
[2025-06-17 15:47:24,763] {bash.py:173} INFO - hey, I am task3 and will be running after task1 at the same time as task2!
[2025-06-17 15:47:24,766] {bash.py:177} INFO - Command exited with return code 0
[2025-06-17 15:47:24,891] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=thrid_task, execution_date=20220117T000000, start_date=20250617T154723, end_date=20250617T154724
[2025-06-17 15:47:25,676] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-17 15:47:25,788] {local_task_job.py:146} INFO - Task exited with return code 0
